qutebrowser/config/configdata.py

```diff
@@ -1239,6 +1239,8 @@ def data(readonly=False):
         ('stop', ['<Ctrl-s>']),
         ('print', ['<Ctrl-Alt-p>']),
         ('open qute:settings', ['Ss']),
+        ('select-follow', ['<Return>']),
```

Comment:
Hmm, I wonder if binding `<Return>` is a good idea... this means it wouldn't be passed to webpages anymore, which makes me wonder how many people rely on that... I guess trying it is the only way to find out :wink: